docs/Readme_Windows_MSVC.md

```diff
@@ -41,8 +40,8 @@ the `static` versions, and OpenTTD currently needs the following dependencies:
 To install both the x64 (64bit) and x86 (32bit) variants, you can use:
 
 ```ps
-.\vcpkg install freetype:x64-windows-static liblzma:x64-windows-static libpng:x64-windows-static lzo:x64-windows-static zlib:x64-windows-static
-.\vcpkg install freetype:x86-windows-static liblzma:x86-windows-static libpng:x86-windows-static lzo:x86-windows-static zlib:x86-windows-static
```

Comment:
Do we want to update the CI to remove this as well?

Reply:
Probably a good idea to check the CI can build without Freetype as well.

Reply:
CI should already build this PR without Freetype as it's removed from the project files, and I tried locally after removing Freetype via vcpkg and it builds fine. But mingw will still build with Freetype as the detection code is untouched.

Reply:
Windows dependencies come in via OpenTTD/CompileFarm, can't change that in this PR.